src/index.js

```diff
@@ -23,18 +23,18 @@ class Service {
   extend(obj) {
     return Proto.extend(obj, this);
   }
-  
+
   _find(params, getFilter = filter) {
     let where = utils.getWhere(params.query);
     let filters = getFilter(where);
     let order = utils.getOrder(filters.$sort);
-    let query = {
+    let query = Object.assign({
       where, order,
       limit: filters.$limit,
       offset: filters.$skip,
       attributes: filters.$select || null
-    };
-    
+    }, params.sequelize);
```

Comment:
:+1: can we put this under `$sequelize` instead? That way it follows a similar pattern to the other special query string params?